manual_jb/content/analyses/housing_household_recovery.md

```diff
@@ -32,11 +32,14 @@ key name | type | name | description
 
 **Input datasets**
 
-key name | type | name | description
---- | --- | --- | ---
-`population_dislocation_block` <sup>*</sup> | [`incore:popDislocation`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:popDislocation) | Population dislocation | Population dislocation results.
-`tpm` <sup>*</sup> | [`incore:houseRecTransitionProbMatrix`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecTransitionProbMatrix) | Probability matrix | A transition probability matrix that specifies<br>the corresponding Markov chain per social vulnerability level.
-`initial_stage_probability` <sup>*</sup> | [`incore:houseRecInitialStageFactors`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecInitialStageFactors) | Mass probability | Initial mass probability function for stage 0 of the Markov Chain.
+key name | type | name                        | description
+--- | --- |-----------------------------| ---
+`population_dislocation_block` <sup>*</sup> | [`incore:popDislocation`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:popDislocation) | Population dislocation      | Population dislocation results.
+`tpm` <sup>*</sup> | [`incore:houseRecTransitionProbMatrix`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecTransitionProbMatrix) | Probability matrix          | A transition probability matrix that specifies<br>the corresponding Markov chain per social vulnerability level.
+`initial_stage_probability` <sup>*</sup> | [`incore:houseRecInitialStageFactors`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:houseRecInitialStageFactors) | Mass probability            | Initial mass probability function for stage 0 of the Markov Chain.
```

Comment:
This should be "initial_stage_probabilities" to match what's in the analysis spec

Reply:
Thanks @navarroc I have addressed both